### DISCUSSION TITLE
1963 Monastic Office?

### DISCUSSION THREAD

**Discussion by Sarto2** [DivinumOfficium/divinum-officium#2583] [Ideas]
Is there a way to implement the Monastic Divine Office according to the Benedictine Use of 1963? Since the website offers a "Pre-Tridentine" version, how about the latest version before the liturgical revolution?

I would be willing to contribute to it the texts since I have access to a copy of the 1963 books (Marietti it is, I think), which are authoritative.

**Comment by APMarcello3** [comment]
We would be delighted to incorporate this version. Since we unfortunately do not have access to the texts that Laszlo Kiss used for the "Pre-Tridentine Monastic" version, it would probably be best to simply transform the files in the "Monastic" version into the 1963 Benedictine usage and adapt them accordingly. Please email me for details.

**Comment by ampmep** [comment]
i would be interrested also and ready to contribute...
monsieur Petit, mep.

Le lun. 16 sept. 2019 à 02:35, Albert P. Marcello, III <
notifications@github.com> a écrit :

> Closed #1256
> <https://github.com/DivinumOfficium/divinum-officium/issues/1256>.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DivinumOfficium/divinum-officium/issues/1256?email_source=notifications&email_token=ALF77K5AFO5O4MP5DHHGW5TQJ3IFDA5CNFSM4IW3XMT2YY3PNVWWK3TUL52HS4DFWZEXG43VMVCXMZLOORHG65DJMZUWGYLUNFXW5KTDN5WW2ZLOORPWSZGOTUHIIPQ#event-2634974270>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALF77K5HU2SV2UFOEGQ4XC3QJ3IFDANCNFSM4IW3XMTQ>
> .
>


-- 
monsieur Petit, mep.
antoine-marie.petit@missionsetrangeres.com
antoinemarie.petit@gmail.com

**Comment by mbab** [comment]
https://github.com/mbab/divinum-officium/tree/mMonastic
is a branch which replaces old _Monastic_ with new _Monastici 1963_.

Changes:
Various changes in code.
New calendar.
Sancti January-March (Latin only, all apart Matutinum as I have only _The Monastic Diurnal_.).

@APMarcello3 please give me know if You are interested in such changes

**Comment by Sarto2** [comment]
Since this thread is open again, I would like to offer my help again.

I have the Antiphonale Monasticum from 1939 with the changes from 1963. I am planning to buy the latest Diurnale Monasticum.

I don't have a single clue about programming or whatever, so I am just going to post the Calendarium in this comment for further implication:

JANUARIUS
01 OCTAVA NATIVITATIS DOMINI I cl
04 S Titi Ep et Conf Memoria
06 IN EPIPHANIA DOMINI I cl
10 S Pauli primi Eremitae Conf. Memoria
13 IN COMMEMORATIONE BAPTISMATIS DNJC II cl
14 S Hilarii Ep. Conf. et Eccl. Doct. Memoria
Commemoratio S Felicis Presb. et Mart.
16 S. Marcelli I Papae et Mart. Memoria
17 S. Antonii Abb. III cl
19 Ss Marii, Marthae, Audifacis et Abachum Mm. Memoria
20 Ss. Fabiani Papae et Sebastiani Mm. III cl
21 S Agnetis Virg. et Mart. III cl
Apud moniales et sorores ON II cl
22 S Vincentii Mart. Memoria
23 S Emerentianae Virg. et Mart. Memoria
24 S Timothei Ep. et Mart. Memoria
25 In Conversione S. Pauli Ap. III cl
Commemoratio S Petri Ap.
26 S Polycarpi Ep. et Mart. Memoria
27 S Ioannis Chrysostomi Ep. Conf. et Eccl. Doct. III cl
28 S Cyrilli Ep. Alex., Conf. et Eccl. Doct. Memoria
29 S. Francisci Salesii Ep. Conf. et Eccl. Doct. Memoria
31 S. Ioannis Bosco Conf. Memoria

FEBRUARIUS
01 S Ignatii Ep. et Mart. III cl
02 IN PURIFICATIONE BMV II cl
03 S Blasii Ep. et Mart. Memoria
05 S Agathae Virg. et Mart. III cl
07 S Romualdi Abb ON III cl
10 S Scholasticae Virg., Sororis SPN Benedicti II cl
Apud moniales et sorores ON I cl
14 S Valentini Presb. et Mart. Memoria
22 Cathedra S Petri Ap. III cl
Commemoratio S Pauli Ap.
23 S Petri Damiani Ep., Conf. et Eccl. Doct. Memoria
24 S Matthiae Ap. II cl
In anno bissextili S Matthiae die 25

MARTIUS
06 Ss Perpetuae et Felicitatis Mm. III cl
07 S Thomae de Aquino Conf. et Eccl. Doct. III cl
09 S Franciscae Romanae Vid. ON Memoria
10 Ss Quadraginta Martyrum Memoria
12 S Gregorii I Papae, Conf. et Eccl. Doct. ON II cl
18 S Cyrilli Ep. Hierosolymitani Conf. et Eccl. Doct. Memoria
19 S IOSEPH SPONSI BMV Conf. et Eccl. univ. Patroni I cl
21 SPN BENEDICTI ABBATIS I cl
25 IN ANNUNTIATIONE BMV I cl
27 S Ioannis Damasceni Conf. et Eccl. Doct. Memoria

APRILIS
04 S Isidori Ep., Conf. et Eccl. Doct. Memoria
11 S Leonis I Papae, Conf. et Eccl. Doct. III cl
13 S Iustini Mart Memoria
14 Ss Tiburtii, Valeriani et Maximi Mm. Memoria
21 S Anselmi Ep., Conf. et Eccl. Doct. III cl
23 S Georgii Mart. Memoria
25 Litania maior
S MARCI EVANGELISTAE II cl
26 S Cleti Papae et Mart. Memoria
27 S Petri Canisii Conf. et Eccl. Doct. Memoria
29 Ss Odonis, Maioli, Odilonis et Hugonis Abb. ON III cl
30 S Catharinae Senensis Virg. Memoria

MAJUS
01 S IOSEPH OPIFICIS SPONSI BMV Conf. I cl
02 S Athanasii Ep., Conf. et Eccl. Doct. III cl
03 Ss Alexandri, Eventii et Theoduli Mm. Memoria
04 s Monicae Vid. Memoria
05 S Pii V Papae et Conf. Memoria
09 S Gregorii Nazianzeni Ep., Conf. et Eccl. Doct. III cl
10 Ss Gordiani et Epimachi Mm. Memoria
11 Ss PHILIPPI ET IACOBI APP. II cl
12 Ss. Nerei, Achillei et Pancratii Mm. Memoria
13 S Roberti Bellarmino Ep. Conf. et Eccl. Doct. Memoria
14 S Pachomii Abbatis Memoria
19 S Petri Caelestini Papae et Conf. ON Memoria
25 S Gregorii VII Papae et Conf. ON III cl
26 S. Augustini Ep. et Conf. ON III cl
27 S Bedae Venerabilis Conf. et Eccl. Doct. III cl

JUNIUS
02 Ss Marcellini et Petri Mm. Memoria
05 S Bonifatii Ep. et Mart. ON III cl
06 S Norberti Ep. et Conf. III cl
09 Ss Primi et Feliciani Mm Memoria
11 S Barnabae Ap. III cl
13 S Antonii de Padua Conf. et Eccl. Doct. Memoria
14 S Basilii Magni Ep., Conf. et Eccl. Doct. III cl
15 S Viti Mart. Memoria
18 S Ephraem Syri Diaconi Conf. et Eccl. Doct. Memoria
Commemoratio Ss. Marci et Marcelliani Mm.
19 Ss Gervasii et Protasii Mm. Memoria
21 S Aloisii Gonzagae Conf. Memoria
23 Vigilia II cl
24 IN NATIVITATE S IOANNIS BAPTISTAE I cl
26 Ss Ioannis et Pauli Mm. Memoria
28 Vigilia II cl
29 SS. PETRI ET PAULI APP. I cl
30 In Commemoratione S Pauli Ap III cl
Commemoratio S Petri Ap.

JULIUS
02 IN VISITATIONE BMV II cl
03 Ss Processi et Martiniani Mm Memoria
11 IN SOLEMNITATE SPN BENEDICTI ABB. II cl
12 S Ioannis Gualberti Abb. ON Memoria
14 S Bonaventurae Ep., Conf. et Eccl. Doct. III cl
17 S Leonis IV Papae et Conf ON Memoria
19 S Vincentii a Paulo Conf. Memoria
20 Ss Hieronymi Aemiliani, Iosephi Calasanctii et Ioannis Baptistae de la Salle Cc. Memoria
22 S Mariae Magdalenae Poenitentis III cl
23 S Apollinaris Ep. et Mart. Memoria
25 S IACOBI AP II cl
26 Ss Ioachim et Annae Parentum BMV III cl
29 Ss Felicis, Simplicii, Faustini et Beatricis Mm. Memoria
30 Ss Abdon et Sennen Mm. Memoria
31 S Ignatii Conf. III cl

AUGUSTUS
01 Ss Machabaeorum Mm. Memoria 
02 S Alfonsi Mariae de Ligorio Ep., Conf. et Eccl. Doct. Memoria
04 S Dominici Conf. III cl
05 In Dedicatione S Mariae ad Nives Memoria
06 IN TRANSFIGURATIONE DNJC II cl
07 Ss Xysti II Papae, Felicissimi et Agapitis Mm. Memoria
08 S Cyriaci Mart Memoria
09 Vigilia III cl
10 S LAURENTII MART II cl
11 S Tiburtii Mart. Memoria
12 S Clarae Virg. Memoria
13 Ss Pontiani Papae et Hippolyti Mm. Memoria
14 Vigilia II cl
15 IN ASSUMPTIONE BMV I cl
18 S Agapiti Mart Memoria
20 S Bernardi Abb., Conf. et Eccl. Doct. ON III cl
21 S Bernardi Ptolomaei Abb. Memoria
22 S Timothei Mart. Memoria
24 S BARTHOLOMAEI AP II cl
28 S Augustini Ep., Conf. et Eccl. Doct. III cl
Commemoratio S Hermetis Mart.
29 In Decollatione S Ioannnis Baptistae III cl
30 Ss Felicis et Audacti Mm Memoria

SEPTEMBER
03 S Pii X Papae et Conf. III cl
08 IN NATIVITATE BMV II cl
09 S Gorgonii Mart. Memoria
11 Ss Proti et Hyacinthi Mm Memoria
14 IN EXALTATIONE S CRUCIS II cl
15 Septem Dolorum BMV III cl
16 Ss Cornelii Papae et Cypriani Ep. Mm. Memoria
17 S Hildegardis Virg. ON Memoria
21 S MATTHAEI AP ET EVANGELISTAE II cl
22 Ss Mauritii et Sociorum Mm. Memoria
23 S Lini Papae et Mart. Memoria
27 Ss Cosmae et Damiani Mm. Memoria
29 IN DEDICATIONE S MICHAELIS ARCHANGELI I cl
30 S Hieronymi Presb., Conf. et Eccl. Doct. III cl

OCTOBER
02 Ss Angelorum Custodum III cl
03 S Teresiae a Iesu Infante Virg. Memoria
04 S Francisci Conf. III cl
05 Ss Mauri et Placidi, discipulorum SPN Benedicti Cc III cl
06 S Brunonis Conf. Memoria
07 BMV a Rosario III cl
14 S Callisti I Papae et Mart. Memoria
15 S Teresiae Virg. III cl
18 S LUCAE EVANGELISTAE II cl
21 S Hilarionis Abbatis Memoria
25 Ss Chrysanthi et Dariae Mm. Memoria
28 SS. SIMONIS ET IUDAE APP II cl

Dominica ultima Octobris: DNIC REGIS I cl

NOVEMBER
01 OMNIUM SANCTORUM I cl
02 IN COMMEMORATIONE OMNIUM FIDELIUM DEFUNCTORUM I cl
04 S Caroli Ep. et Conf. Memoria
08 Ss Quatuor Coronatorum Mm. Memoria
09 IN DEDICATIONE ARCHIBASILICAE SS.MI SALVATORIS II cl
10 S Theodori Mart Memoria
11 S Martini Ep. et Conf. II cl
12 S Mennae Mart. Memoria
13 OMNIUM SANCTORUM ON II cl
14 IN COMMEMORATIONE OMNIUM DEFUNCTORUM ON II cl
vel de ea (IV cl)
15 S Alberti Magni Ep., Conf. et Eccl. Doct. Memoria
17 S Gertrudis Virg. ON III cl
Apud moniales et sorores ON II cl
18 In Dedicatione Basilicarum Ss. Petri et Pauli App. III cl
21 In Praesentatione BMV III cl
Commemoratio S Columbani Abb.
22 S Caeciliae Virg. et Mart. III cl
23 S Clementis I Papae et Mart. III cl
Commemoratio S Felicitatis Mart.
24 S Ioannis a Cruce Conf. et Eccl. Doct. Memoria
Commemoratio S Chrysogoni Mart.
25 S Catharinae Virg. et Mart. Memoria
26 S Silvestri Abb. ON Memoria
29 S Saturnini Mart Memoria
30 S ANDREAE AP II cl

DECEMBER

02 S Petri Chrysologi Ep., Conf. et Eccl. Doct. Memoria
03 S Francisci Xaverii Conf. Memoria
06 S Nicolai Ep. et Conf. Memoria
07 S Ambrosii Ep., Conf. et Eccl. Doct. III cl
08 IN CONCEPTIONE IMMACULATA BMV I cl
11 S Damasi I Papae et Conf. Memoria
13 S Luciae Virg. et Mart. III cl
21 S THOMAE AP II cl
24 Vigilia I cl
25 IN NATIVITATE DOMINI I cl cum Octava
In secunda Missa: Commemoratio S Anastasiae Mart.
26 II dies infra octavam Nativitatis Domini. S. STEPHANI PROTOMART. II cl
27 III dies infra octavam Nativitatis Domini. S IOANNIS AP ET EVANGELISTAE II cl
28 IV dies infra octavam Nativitatis Domini. SS INNOCENTIUM MARTYRUM II cl
29 DE V DIE INFRA OCTAVAM NATIVITATIS DOMINI II cl
30 DE VI DIE INFRA OCTAVAM NATIVITATIS DOMINI II cl
31 DE VII DIE INFRA OCTAVAM NATIVITATIS DOMINI II cl
Commemoratio S Silvestri I Papae et Conf.

**Comment by APMarcello3** [comment]
@mbab Please proceed with transforming "Monastic" into the 1963 Breviarium Monasticum.

**Comment by APMarcello3** [comment]
Cf. also issue #1066.

**Comment by mbab** [comment]
My Diurnal has Calendar approved July 28, 1961, should we stick with 1963 or 1961?

**Comment by Sarto2** [comment]
The approbation printed in my Antiphonale dates to July 25, 1961 (a SRC approbatum). If they haven't changed the Calendar in the 1963 Marietti Breviary, then those should all conform, no matter the year.

**Comment by APMarcello3** [comment]
I think any differences between 1961 and 1963 would be trivial; if there are any, I am sure they can be rectified without much problem.

**Comment by APMarcello3** [comment]
As this project is well underway, I feel confident to close this issue.

**Comment by mbab** [comment]
I'm happy to inform You that Monastic version is now updated to 1960.

So I'm inviting You to take part in vernaculars translations.
(There are few offices which need English translations!)
